src/backend/Core/Database/Seeders/WarehouseSeeder.cs

```diff
@@ -0,0 +1,213 @@
+﻿using Core.Database.Enums;
+using Core.Database.Models.Auth;
+using Core.Database.Models.Core;
+using Microsoft.EntityFrameworkCore;
+
+namespace Core.Database.Seeders
+{
+    public static class WarehouseSeeder
+    {
+        public static void Execute(ModelBuilder builder)
+        {
+            builder.Entity<Warehouse>().HasData(new Warehouse()
+            {
+                Id = 1,
+                Status = DbEntityStatus.Active,
+                UpdatedAt = new DateTime(2023, 01, 01),
```

Comment:
Typ daty powinien być UTC, we wszystkich przypadkach tutaj -> `new DateTime(2023, 01, 01, 0, 0, 0, DateTimeKind.Utc)`